src/raw/mod.rs

```diff
@@ -4280,6 +4280,7 @@ impl ExactSizeIterator for FullBucketsIndices {}
 impl FusedIterator for FullBucketsIndices {}
 
 /// Iterator which consumes a table and returns elements.
+#[derive(Clone)]
 pub struct RawIntoIter<T, A: Allocator = Global> {
     iter: RawIter<T>,
     allocation: Option<(NonNull<u8>, Layout, A)>,
```

Comment:
This is not going to work correctly since it will end up double-freeing the underlying allocation.

Reply:
Right, that makes sense.

I'll look into actually cloning the allocation instead.